src/pages/[id]/index.js

```diff
@@ -74,7 +73,7 @@ export async function getServerSideProps(context) {
     }
     const { user } = await res.data;
     const getImageLink = (transformString) => {
-      return !!dev && user.picture
+      return user.picture
         ? getCloudinaryImgURL(user.picture.publicId, transformString)
         : getImgURL(user.username, 'img.png');
```

Comment:
Can we remove this function call? We are planning on deprecating this.

Reply:
made the necessary changes,